### PR TITLE
clt

### PR DESCRIPTION
**Pull request by Daniel-Zaretsky**
@guyfarhi11 @shai-shechter 
Karin isn't here so I tagged you.
This is the clt file for Guy Farhi and Daniel Zaretsky

### CODE REVIEW

File: users/daniel.zaretsky/assignment1/clt.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        Parameters:
+            S (pandas series): data to be sampled.
+            n (natural number): number of samples per mean.
+        Returns:
+            Pandas series: A series of means of samples.
+    """
+    import random
+    random.seed(99)
+    k = 1000
+    means = []
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:14]: 'means' is unnecessary here (you don't use it)

File: users/daniel.zaretsky/assignment1/clt.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return S.sample(n * k, replace = True).values.reshape(k, n).mean(axis = 1)
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:15]: Great implementation! :)

File: users/daniel.zaretsky/assignment1/clt.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+def hist_means(S, n):
+    """
+    The function takes a pandas series S and a natural number n and plots a histogram of the series of means, and also indicates the mean of this series.
+
+    Parameters:
+        S (pandas series): data to be sampled.
+        n (natural number): number of samples per mean.
+    """
+    sample = means_of_samples(S, n)
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:25]:
```suggestion
    means_series = means_of_samples(S, n)
```

File: users/daniel.zaretsky/assignment1/clt.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    import matplotlib.pyplot as plt
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:26]: same comments of imports

File: users/daniel.zaretsky/assignment1/clt.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    plt.hist(sample)
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:27]:
```suggestion
    plt.hist(means_series)
```

File: users/daniel.zaretsky/assignment1/clt.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    plt.xlabel('Activation probability')
+    plt.ylabel('Counts')
+    plt.title(f'Mean: {sample.mean():.3f}, Sample size: {n}')
```
Comment (karin-cohen) [inline users/daniel.zaretsky/assignment1/clt.py:30]:
```suggestion
    plt.title(f'Mean: {means_series.mean():.3f}, Sample size: {n}')
```

**Comment by Daniel-Zaretsky** [comment]
@karin-cohen Fixed it 😁  thx

**Pull request closed without merging**